packages/analytics-browser/src/config.ts

```diff
@@ -244,6 +245,7 @@ export const useBrowserConfig = async (
     sessionId,
     options.sessionTimeout,
     options.storageProvider,
+    options.savedMaxCount,
```

Comment:
do we need to update the params in `BrowserConfig`?

Reply:
Put `savedMaxCount` in `Storage`. Please see the latest commits.